packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -434,25 +516,35 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
                   );
                 }
               }
-              mutateCurrentVisit();
-              mutateVisits();
-              closeWorkspace({ ignoreChanges: true });
 
-              showSnackbar({
-                isLowContrast: true,
-                timeoutInMs: 5000,
-                kind: 'success',
-                subtitle: !visitToEdit
-                  ? t('visitStartedSuccessfully', '{{visit}} started successfully', {
-                      visit: response?.data?.visitType?.display ?? t('visit', 'Visit'),
-                    })
-                  : t('visitDetailsUpdatedSuccessfully', '{{visit}} updated successfully', {
-                      visit: response?.data?.visitType?.display ?? t('pastVisit', 'Past visit'),
-                    }),
-                title: !visitToEdit
-                  ? t('visitStarted', 'Visit started')
-                  : t('visitDetailsUpdated', 'Visit details updated'),
-              });
+              from(handleVisitAttributes(visitAttributes, response.data.uuid))
```

Comment:
Why are we using `rxjs` here instead of, e.g., `then()` or `await`?

Reply:
Hello @ibacher , I actually had the `await` first and then using `rxys` was your suggestion [here](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1616#discussion_r1470149105)

Reply:
🤦 Sorry for the whiplash. Go with what older me said.